middleware/router.go

```diff
@@ -53,9 +53,15 @@ func NewRouter(router *gin.Engine) *gin.Engine {
 
 	// HTML Template generated pages
 	router.GET("/status", httpTemplates.StatusPageResponse(), StatusPageMiddleware())
+	router.GET("/console", ConsolePageMiddleware())
```

Comment:
What the console means? Is this own account cabinet?

Reply:
Didn't know how to name it. The console is something like account cabinet yep, but in case of framework it's just sign in, sign up and logout buttons